walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -270,5 +282,52 @@ public void notEnoughRoom() {
 
     Assert.assertEquals(expected, actual);
   }
+
+  // 6/18 Test for showing time slots for an event with two optional attendees wtih sparse available slots 
+  @Test
```

Comment:
For these test cases, can you also add a diagram like the one above to make it easier to visualize what the test case should be doing.
```
    // Have one person, but make it so that there is not enough room at any point in the day to
    // have the meeting.
    //
    // Events  : |--A-----| |-----A----|
    // Day     : |---------------------|
    // Options :
```

Reply:
Gotcha; will do that